src/Illuminate/Console/Application.php

```diff
@@ -102,6 +112,28 @@ public function call($command, array $parameters = [])
     }
 
     /**
+     * Runs the command container.
+     *
+     * @param  InputInterface  $input  An Input instance
+     * @param  OutputInterface $output An Output instance
+     * @return int
+     */
+    public function run(InputInterface $input = null, OutputInterface $output = null)
+    {
+        $this->events->fire(
+            new Events\CommandStarting($this->getCommandName($input), $input)
```

Comment:
`$this->getCommandName($input)` is repeated twice and could be extracted to `$commandName`